src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,19 @@
+.catalog {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
```

Comment:
The grid-template-columns property is set to a fixed width of 200px. Consider using a more flexible unit like 'fr' or percentages to allow the grid to adapt better to different screen sizes.